python/paddle/tensor/einsum.py

```diff
@@ -664,7 +666,139 @@ def plan_einsum(operands, g_view, g_shape, g_supports, g_count, n_bcast):
     return plan
 
 
+def preprocess(equation, *operands):
+    """
+    check equation / raise error, default right labels generation
+    """
+    equation = equation.replace(" ", "")
+    nop = len(operands)
+    assert nop > 0, "At least one operand is expected."
+
+    # Part the equation to left hand side and right hand side
+    lhs, *rhs = equation.lower().replace(' ', '').split('->')
```

Comment:
```suggestion
    lhs, *rhs = equation.lower().split('->')
```
前面 equation 已经替换过空格了